pkg/factories/command_factory.go

```diff
@@ -53,10 +53,10 @@ func (f *kafkaSourceCommandFactory) SourceCommand() *cobra.Command {
 
 func (f *kafkaSourceCommandFactory) CreateCommand() *cobra.Command {
 	createCmd := f.defaultCommandFactory.CreateCommand()
-	createCmd.Use = "create NAME --servers SERVERS --topics TOPICS --consumergroup GROUP --sink SINK"
+	createCmd.Use = "create NAME --servers SERVERS --topics TOPICS --consumergroup GROUP --sink SINK --ce-override CE-OVERRIDE"
```

Comment:
I'd suggest to keep main use example only with required parameters. Since `ce-override` is optional it's better to just keep in in the example below. E.g. add a new line example.

Reply:
With regard to the above comment I'd remove `--consumergroup` as well.